sonar-batch/src/main/java/org/sonar/batch/sensor/DefaultSensorStorage.java

```diff
@@ -173,8 +172,9 @@ private void setValueAccordingToMetricType(Measure<?> measure, org.sonar.api.mea
       case DISTRIB:
         measureToSave.setData((String) measure.value());
         break;
+      case MILLISEC:
       case WORK_DUR:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This case's code block is the same as the block for the case on line 161. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1871)